src/ContaoManager/Plugin.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2016 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\FaqBundle\ContaoManager;
+
+use Contao\ManagerBundle\ContaoManager\Bundle\BundleConfig;
+use Contao\ManagerBundle\ContaoManager\Bundle\BundlePluginInterface;
+use Contao\ManagerBundle\ContaoManager\Bundle\ParserInterface;
+
+/**
+ * Plugin for the Contao Manager.
+ *
+ * @author Andreas Schempp <https://github.com/aschempp>
+ */
+class Plugin implements BundlePluginInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public function getBundles(ParserInterface $parser)
+    {
+        return [
```

Comment:
Ok. Would be nice (and probably less error prone) if it supported `@return ConfigInterface| ConfigInterface[]`, so either a single configuration or an array, but I can also live without it.